### PR TITLE
C116 styling

### PR DESCRIPTION
**Pull request by HadarRose**
- Added CSS file
- removed certain styling components from basicRequests.js

### CODE REVIEW

File: src/css/index.css [hunk 1]
```diff
@@ -0,0 +1,103 @@
+/* TODO delete this probably
+:root{
+    --title-margin: 20;
+}*/
+
+* {
+    font-family: 'Times New Roman', Times, serif;
+}
+
+body {
```
Comment (kev-chan) [inline src/css/index.css]: don't use tag names unless you absolutely need to

File: src/css/index.css [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    margin: 60px;
+    margin-top: 0px;
+}
+
+button {
```
Comment (kev-chan) [inline src/css/index.css]: same as above

File: src/css/index.css [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+:root{
+    --title-margin: 20;
+}*/
+
+* {
+    font-family: 'Times New Roman', Times, serif;
+}
+
+body {
+    margin: 60px;
```
Comment (kev-chan) [inline src/css/index.css]: is there a better way to write this and the line below?

File: src/css/index.css [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+.title-div {
+    text-align: center;
+    font-size: 40pt;
+    font-weight: bold;
+    margin-top: 25px;
+}
+
+.timeline-container {
+    border-style: solid;
```
Comment (kev-chan) [inline src/css/index.css]: single line border

File: src/css/index.css [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+.timeline-container > div.tweet-block:nth-of-type(odd){
+    background-color: #e8f5fd;
+}
+
+.timeline-container > div.tweet-block:nth-of-type(even){
+    background-color: #e9e9e9;
+}
+
+.timeline-container > div.tweet-block:last-child{
+    border-bottom-style: none;
```
Comment (kev-chan) [inline src/css/index.css:45]: nice, that works

File: src/css/index.css [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+}
+
+.tweet-block{
+    border-bottom-style: solid;
```
Comment (kev-chan) [inline src/css/index.css]: same as above

File: src/css/index.css [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+    font-size: 75%;
+}
+
+.message-div > .date{
+    font-style: italic;
+    font-size: 80%;
+    padding-bottom: 5px;
+}
+
+.message-div > a{
```
Comment (kev-chan) [inline src/css/index.css]: same as above regarding tag names

File: src/js/basicRequests.js [hunk 8]
```diff
@@ -26,37 +30,53 @@ function getTimeline() {
...
+                    // add div to container
+                    div.className = "tweet-block";
                     container.appendChild(div);  
                 });
             }
         }
     };
     xhttp.onerror = function(){ 
-        document.getElementById("timeline-container").innerHTML = "Something went wrong, please contact systems administrator.";
+        let errorDiv = document.createElement("div");
```
Comment (kev-chan) [inline src/js/basicRequests.js]: spacing but this could be combined with the above code as well

File: src/css/index.css [hunk 9]
```diff
@@ -0,0 +1,104 @@
+/* TODO delete this probably
```
Comment (kev-chan) [inline src/css/index.css]: if this isn't needed, just delete vs leaving TODO

File: src/css/index.css [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+:root{
+    --title-margin: 20;
+}*/
+
+* {
```
Comment (kev-chan) [inline src/css/index.css]: this works but better to wrap it in an identifier

File: src/css/index.css [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+.timeline-container > div.tweet-block:last-child{
+    border-bottom-style: none;
+}
+
+.tweet-block{
+    border-bottom-style: solid;
+    border-color: #c3c3c4;
+    border-width: 1pt;
+    padding: 20px;
+    padding-bottom: 10px;
```
Comment (kev-chan) [inline src/css/index.css]: is there a different way to write this and the line above it?

**Review by kev-chan — CHANGES_REQUESTED** [review]

File: src/css/index.css (hunk 4 above)
Comment (HadarRose) [reply src/css/index.css]: Could you clarify this comment please?

File: src/css/index.css (hunk 10 above)
Comment (HadarRose) [reply src/css/index.css]: Just for clarification, you are talking about the '*' which changes the font-family, and not the parameters in the comments I forgot to delete, correct?

File: src/css/index.css (hunk 4 above)
Comment (HadarRose) [reply src/css/index.css]: Oh, do you mean writing the shorthand border in one line?

File: src/css/index.css (hunk 10 above)
Comment (kev-chan) [reply src/css/index.css]: yeah

**Pull request merged**